### PR TITLE
Add specviz plot settings to rtd

### PR DESCRIPTION
**Pull request by javerbukh**
Adds documentation and an image to the `Plot Settings` section of the Specviz rtd page.

### CODE REVIEW

File: docs/specviz/displaying.rst [hunk 1]
```diff
@@ -24,4 +24,26 @@ More words...
...
+
+Layer
+-----
+
+The top section of the "Layer" tab contains options to change the color of the spectrum (click the "square" icon to see a color change menu), change visibility of the spectrum ("eye" icon), and a drop-down box to select which layer will have its' settings changed.
+
+Line Width
+----------
+
+Pixel width of the spectra. Larger values correspond to thicker lines on the plot.
```
Comment (rosteen) [inline docs/specviz/displaying.rst:39]:
```suggestion
Width of the spectrum in pixels. Larger values correspond to thicker lines on the plot.
```

File: docs/specviz/displaying.rst [hunk 2]
```diff
@@ -24,4 +24,26 @@ More words...
...
+
+Vmin and Vmax
+-------------
+
+Minimum and maximum values of the y axis.
+
+Percentile
+----------
+
+Include data within this percentile in the viewer.
```
Comment (rosteen) [inline docs/specviz/displaying.rst:49]:
```suggestion
Sets the bounds of the plot (Vmin and Vmax) such that the selected percentage of the data is shown in the viewer. Editing either bound manually changes the "Percentile" selection to "Custom."
```

**Comment by javerbukh** [comment]
@rosteen Implemented, thank you!

File: docs/specviz/displaying.rst [hunk 3]
```diff
@@ -24,4 +24,26 @@ More words...
 Plot Settings
 =============
 
-More words...
\ No newline at end of file
+To access plot settings for a particular viewer (including the spectrum viewer), click the "hammer and screwdriver" icon, followed by the "gear" icon, followed by the "Layer" tab.
```
Comment (PatrickOgle) [inline docs/specviz/displaying.rst:27]:
```suggestion
To access plot settings for a particular viewer (including the spectrum viewer), click the :guilabel:`hammer and screwdriver` icon, followed by the :guilabel:`gear` icon, followed by the  :guilabel:`Layer` tab.
```

File: docs/specviz/displaying.rst [hunk 4]
```diff
@@ -24,4 +24,26 @@ More words...
...
+
+.. image:: img/specviz_plot_settings.png
+
+Layer
+-----
+
+The top section of the "Layer" tab contains options to change the color of the spectrum (click the "square" icon to see a color change menu), change visibility of the spectrum ("eye" icon), and a drop-down box to select which layer will have its settings changed.
```
Comment (PatrickOgle) [inline docs/specviz/displaying.rst:34]:
```suggestion
The top section of the :guilabel:`Layer` tab contains options to change the color of the spectrum (click the :guilabel:`square` icon to see a color change menu), change visibility of the spectrum (:guilabel:`eye` icon), and a drop-down box to select which layer will have its settings changed.
```

File: docs/specviz/displaying.rst [hunk 5]
```diff
@@ -24,4 +24,26 @@ More words...
...
+
+Vmin and Vmax
+-------------
+
+Minimum and maximum values of the y axis.
+
+Percentile
+----------
+
+Sets the bounds of the plot (Vmin and Vmax) such that the selected percentage of the data is shown in the viewer. Editing either bound manually changes the "Percentile" selection to "Custom."
```
Comment (PatrickOgle) [inline docs/specviz/displaying.rst:49]: I noticed that there is an :guilabel:`up/down' button at the bottom of the Layer tab that will renormalize the spectrum.  I was unaware of it until yesterday.  Do you want to document what it does here?

Comment (PatrickOgle) [reply docs/specviz/displaying.rst:49]: Actually, it is in the Viewer tab, not the layer tab...my bad.

**Comment by javerbukh** [comment]
Merging.

**Pull request merged**